bin/storm.py

```diff
@@ -754,18 +827,24 @@ def parse_config_opts(args):
     elif token == "--config":
       global CONFFILE
       CONFFILE = curr.pop()
+    elif token == "--jars":
```

Comment:
I think --jars is representing its meaning properly so I'd like to keep it as it is.
If there's other one which would want to change option to non-duplicated name, I'll do that.